README.md

```diff
@@ -80,6 +80,7 @@ GCC Fortran | 7, 8, 9, 10 | MacOS Catalina 10.15 | x86_64
 GCC Fortran | 8 | Windows Server 2019 | x86_64
 GCC Fortran (MSYS) | 10 | Windows Server 2019 | x86_64
 GCC Fortran (MinGW) | 10 | Windows Server 2019 | x86_64, i686
+GCC Fortran (MinGW) | 8, 9, 10 | Windows 10 | x86_64, i686
```

Comment:
I propose to move this line an additional table, because these are not tested within the WindowsCI (if I am correct).
Therefore, if a new procedure is added to stdlib, these combinations OS-compiler will not tested for this new procedure.